templates/plugin/amqp1.conf.erb

```diff
@@ -0,0 +1,46 @@
+<Plugin amqp1>
```

Comment:
can you please provide this as an epp and not erb template? We agreed on epp some time ago and documented it in our review guideline at https://voxpupuli.org/docs/#reviewing-a-module-pr

Reply:
Ok, didn't notice this as all templates are in ERB. Will rewrite the template then.